price-matching-system.pyproj

```diff
@@ -28,6 +28,7 @@
     <EnableUnmanagedDebugging>false</EnableUnmanagedDebugging>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="price_matching_system\auth - Copy.py" />
```

Comment:
this seems like a leftover reference to an unused file.
should be deleted.

Reply:
Thanks, a great one.

Reply:
deleted